docs/pages/guide/performance_tips.md

```diff
@@ -0,0 +1,49 @@
+`sklearn-raster` supports using [Dask](https://docs.dask.org/en/stable/) to apply machine learning in parallel. Choosing to [use Dask-backed data](raster_formats.md) instead of Numpy arrays can lead to significant speedups and allow working with large datasets that don't fit into memory, but requires some additional planning to ensure that operations are optimized for your datasets and hardware.
```

Comment:
Check my grammar, but I think it should be "and *allows* working with large datasets ..."